articles/how-to-optimize-wordpress-website-speed/index.md

```diff
@@ -0,0 +1,214 @@
+The ultimate measure of websites performance is loading speed. This is the amount
+of time it takes to load a web page. Web users are relying more on
+mobiles to access web content and services and thus more demand for optimized websites. If you run any website, this
+is very evident in your analytical data. Users are also demanding better web experience thus the need to improve your WordPress website performance. To improve overall website performance, several measures
+need to be taken. This article is a comprehensive guide on the best ways to speed up WordPress website.
+
+## Why Website Speed Matters?
+
+If your website takes longer than 2 seconds to load, the bounce rate increases.
+This means you will have reduced conversions and a decrease in
+revenue. According to the [Google SEO guide](https://www.google.com/search/howsearchworks/algorithms/), "page speed" of sites is among the parameters considered in the search engine ranking algorithm since January 2018. Slower websites are ranked lower as compared to faster loading websites. Fast loading web pages lead to improved user experience, rise in page views, and contribute to higher search engine rankings.
```

Comment:
webpages should be one word not two separate words. Consider omitting loading in fast loading webpages to simplify the sentence.